microsoft-edge/webview2/release-notes.md

```diff
@@ -34,6 +34,225 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 
 To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test upcoming APIs and features](how-to/set-preview-channel.md).
 
+<!-- ====================================================================== -->
+## 1.0.1587.40
+
+Release Date: Feb 13, 2023  
+
+[NuGet package for WebView2 SDK 1.0.1587.40](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1587.40)
+
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 110.1587.40 or higher. 
+
+### General
+
+#### Promotions
+
+The following items are now stable:
+
+*  Additional options used to create WebView2 Environment to manage custom scheme registration.
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2EnvironmentOptions Class](#)
+   * [CoreWebView2EnvironmentOptions.CustomSchemeRegistrations Property](#)
+* [CoreWebView2CustomSchemeRegistration Class](#)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2EnvironmentOptions Class](#)
+* [CoreWebView2CustomSchemeRegistration Class](#)
+

```

Comment:
<!-- todo: is this needed, as the middle list item, like .NET has above?
   * [CoreWebView2EnvironmentOptions.CustomSchemeRegistrations Property](#)
-->